models.py

```diff
@@ -1,30 +1,22 @@
-"""
-Todo:
-Try Adagrad and Adadelta
-Implement cnn+lstm
-Implement FastText embedding + models
-Research/implement Region embeddings from http://proceedings.mlr.press/v48/johnson16.pdf
-Experiment with Attention and Dilated Convolutions
-"""
-
 from keras import regularizers
```

Comment:
@jakubczakon My impression is that `models.py` should have only models classes (CharVDCNN, WordSCNN, etc.). Other definitions should go to the separate, more atomic file, in the `steps` backend, for example, steps -> keras -> architectures.py. These definitions are, i.e.: _dpcnn_block, _dense_block, _classification_block, because they are pieces on the network architecture.

Reply:
@jakubczakon Move lasses with models to the steps -> keras -> models.py.
All functions (functions that makes model and private functions that makes building blocks) goes to steps -> keras -> architectures.py